app/models/review_reminder.rb

```diff
@@ -17,4 +19,10 @@ def review_date_cannot_be_in_the_past
       errors.add(:review_at, "can't be in the past")
     end
   end
+
+  def reset_reminder_sent_at_if_review_at_is_updated
+    return unless changed.include?("review_at") && reminder_sent_at.present?
+
+    self.reminder_sent_at = nil
+  end
```

Comment:
...meaning this can become a one-liner:

```suggestion
  def reset_reminder_sent_at
    self.reminder_sent_at = nil
  end
```

The guard clause is handled by the `if:` config on the `before_update`. And I think the `reminder_sent_at.present?` was probably redundant, because it should be harmless to set the field to `nil` if it's already blank.